### ISSUE TITLE
Unlinking mobs

### ISSUE THREAD

**Issue by hightin** [Tomslack/ManbabyDungeonTools#46]
Could you unlink some packs? I can update it on my end but it reverts every time I update the addon itself. I don't know how to send you the edited version or I would go through and do it again myself and send it to you, sorry new to GitHub.

Halls of Attonement:
These mobs should be linked together and not in the group they are currently in:
Shard of Halkias 1 (G12)
Depraved Collector 1 (G12)

Shard of Halkias 3 (G3)
Depraved Collector 3 (G3)

Shard of Halkias 2 (G25)
Depraved Collector 2 (G25)

These mobs should be unlinked from eachother:
Stoneborn Slasher 2 (G38)
Stoneborn Slasher 4 (G38)

De Other Side:
These mobs should be unlinked from their groups:
4.RF-4.RF 1 (G8)
Headless Client 1 (G8)
Territorial Bladebeak 1, 3, 4, 5, 6, 8, 9, 10, 11, 12
Bladebeak Matriarch 3 (G34)

Plaguefall:
These mobs should be unlinked from their groups:
Plagueborer 2 (G5) - This pats away from its pack and can be solo pulled then.
Plagueborer 3 (G6) - This pats away from its pack and can be solo pulled then.

Sanguine Depths:
These mobs should be unlinked from their groups:
Gluttonous Tick 1 (G4)
Gluttonous Tick 2 (G4)
Gluttonous Tick 3 (G4)
Grubby Dredger 1 (G4)
Grubby Dredger 2 (G4)
Insatiable Brute 2 (G4)
Chamber Sentinel 5 (G34)

Theater of Pain:
These mobs should be unlinked from their groups:
Shattered Souls 1-12

**Comment by srenauld** [comment]
Sure can!

I'll go and test on my hunter tomorrow, just to make sure it's actually correct before releasing. Guilty, Jarx and I were planning to sweep all dungeons for this stuff.

If you see anything else, don't hesitate to edit or comment here :-)

**Comment by jarxjarx** [comment]
IS the G34 for sanguine really accurate? If you pull the chamber sentinel the mistdancer should come, but the overseer doesn't.
G21 in spires are also not fully linked, the 3 etherdivers are linked, but they're not linked to the goliath.

**Comment by hightin** [comment]
I don't know about pulling the Mistdancer but I know for sure if you pull the Overseer then the Sentinel does not come with. I taunt pull him every run cause that is where I spawn the pride, pulling the Overseer and killing him and the Mistdancer spawns pride.

Also in Sanguine there seems to be missing patrols (G22 I think and G29).

True about G21 in Spires, I don't run that side often so I didn't think about it when I was going through the maps.

**Comment by srenauld** [comment]
This is when I wish the marking in the route strings didn't work off groups. Due to this, a ton of routes published out there will lose percentage. Necessary, though.

Verified every single HoA mob, you were right on all of them; I'll think of a good way to make the change while breaking as little as possible.

**Comment by srenauld** [comment]
Just so we're all up to speed, we've split the issue in the following way behind the scenes:
- @jandres61 : Mists
- @srenauld : HoA, Sanguine Depths, Plaguefall
- @Tomslack : General look and see for consistency across dungeons

**Comment by Drangleic** [comment]
Also, I believe in Halls of Atonement the Depraved Obliterators in G 12 and G 25 are should not be linked with the the Depraved Collectors and Shards of Halkias. Unless something has changed it is possible to pull the Obliterators without pulling the Collectors and Shards.

**Comment by srenauld** [comment]
@Drangleic That's correct, but I need to think of a good way to split without breaking a whole bunch of people's routes, or break them as minimally as possible (right now each side is one whole group and as such, people's percentages on their currently-imported route will drop by 3-4% if I just brute-force split it.

The two obliterators on each side are linked together.

The collector is on his own.

The shard is on his own.

It's the same thing, the two slashers in front of Echelon are not linked, but MDT has it linked; this one is a much easier split since most routes don't pull either of them, though!
